src/openforms/config/models.py

```diff
@@ -37,6 +37,27 @@ class GlobalConfiguration(SingletonModel):
         default="Thank you for submitting this form.",
     )
 
+    confirmation_email_subject = models.CharField(
+        _("confirmation email subject"),
+        max_length=1000,
+        help_text=_(
+            "Subject of the confirmation email message. Can be overridden on the form level"
+        ),
+        default="Bevestiging van uw inzending",
+    )
+
+    confirmation_email_content = models.TextField(
+        _("confirmation email content"),
+        help_text=_(
+            "Content of the confirmation email message. Can be overridden on the form level"
+        ),
+        default="""
+        Geachte heer/mevrouw,
+
+        Wij hebben uw inzending, met referentienummer {{ public_reference }}, in goede orde ontvangen.
+        """,
```

Comment:
I'm not sure what we want for a default?

Also I couldn't seem to get the newlines working.  Not sure what the issue is but I'll look more into this once I know what the default will be and make sure it renders correctly.